src/Foundation.Find.Cms/Models/Pages/PersonListPage.cs

```diff
@@ -0,0 +1,16 @@
+﻿using EPiServer.DataAbstraction;
+using EPiServer.DataAnnotations;
+using Foundation.Cms.Pages;
+
+namespace Foundation.Find.Cms.Models.Pages
+{
+    [ContentType(DisplayName = "Person List Page", 
+        GUID = "4f0203b6-d49e-4683-9ce6-ede8c37c77d3", 
+        Description = "Person List Page helps users can find people within an organisation",
```

Comment:
Please follow standard: Capitalisation (first letter in uppercase and the remaining letters in lowercase)